pkgs/by-name/gi/gitlab-release-cli/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitLab
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "gitlab-release-cli";
+  version = "0.18.0";
+
+  src = fetchFromGitLab {
+    owner = "gitlab-org";
+    repo = "release-cli";
+    rev = "v${version}";
+    sha256 = "sha256-CCSice/uMf2OfFNEpwwhX6A0wrSsC1v9XWEhAAwQRso=";
+  };
+
+  vendorHash = "sha256-UwDMRsWbk8rEv2d5FssIzCLby68YZULoxd3/JGLsCQU=";
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Better to avoid it.

Reply:
What's the reasoning, of why this should be avoided?

Reply:
`lib` is a very broad scope, which when mixed with the [inherit issues](https://nix.dev/guides/best-practices#with-scopes) of `with ...;` can lead to a confusion at best and weird variable shadowing at worst. also see https://github.com/NixOS/nixpkgs/issues/208242